### PR TITLE
Adding package python-libsbml-experimental

### PR DESCRIPTION
**Pull request by fbergmann**
This package was requested, as it provides support for experimental extensions to SBML as well. 

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/python-libsbml-experimental/build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+echo "Running build.sh"
+echo "PWD: ${PWD}"
+echo "SRC_DIR: ${SRC_DIR}"
+echo "RECIPE_DIR: ${RECIPE_DIR}"
+echo "PREFIX: ${PREFIX}"
+
+ls -l
```
Comment (scopatz) [inline recipes/python-libsbml-experimental/build.sh:7]: Is this needed?

File: recipes/python-libsbml-experimental/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+{% set name = "python-libsbml-experimental" %}
```
Comment (scopatz) [inline recipes/python-libsbml-experimental/meta.yaml:1]: Why is this called experimental when the package is just `python-libsbml`?

Comment (fbergmann) [reply recipes/python-libsbml-experimental/meta.yaml:1]: the archive contains sources for both packages, an environment variable controls which package is to be built.

File: recipes/python-libsbml-experimental/build.sh (hunk 1 above)
Comment (fbergmann) [reply recipes/python-libsbml-experimental/build.sh]: removed

File: recipes/python-libsbml-experimental/meta.yaml (hunk 2 above)
Comment (scopatz) [reply recipes/python-libsbml-experimental/meta.yaml:1]: Can you please make a note of that in a comment

Comment (fbergmann) [reply recipes/python-libsbml-experimental/meta.yaml:1]: done

**Comment by bgruening** [comment]
Thanks @fbergmann!

**Pull request merged**